Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -37,8 +37,8 @@
     end
 
     it "returns true when ctime and mtime < days_default" do
-      allow_any_instance_of(Pathname).to receive(:ctime).and_return(2.days.ago)
-      allow_any_instance_of(Pathname).to receive(:mtime).and_return(2.days.ago)
+      allow_any_instance_of(Pathname).to receive(:ctime).and_return((DateTime.now - 2).to_time)
```

Comment:
I prefer this style of subtracting days from a `Time` object (it's less math-y), but there's a different, existing style elsewhere in this file (`Time.now - (2 * 60 * 60 * 24)`). I'm happy to use that one for consistency, if reviewers prefer.

Reply:
I prefer your way, but then I'm allergic to maths. 😉

Reply:
I don't think it matters in this case but I don't think those two options are 100% similar.

```rb
irb(main):009:0> (Time.now - (2 * 60 * 60 * 24))
=> 2023-02-02 18:17:39 -0800
irb(main):010:0> (DateTime.now - 2).to_time
=> 2023-02-02 18:18:00 -0800
```

I prefer the `DateTime` version too.